### DISCUSSION TITLE
How do I prevent auto indent and * with each new line after a /*?

### DISCUSSION THREAD

**Discussion by gaveitatry2** [violentmonkey/violentmonkey#1670] [Q&A]
I am using Violentmonkey 2.13.3 for Mozilla Firefox.

These are my Editor settings:

```
{
  "lineWrapping": true,
  "indentUnit": 0,
  "killTrailingSpaceOnSave": false
}
```

This is what happens when I press Enter after a /*:

https://user-images.githubusercontent.com/75463665/205413763-bf976074-6151-48bc-a72f-9d58504a9aab.mp4

**Comment by tophf** [comment]
Press Shift-Enter to disable it just once or add `"continueComments": false,` to your editor config to disable it entirely.
To see all available options click `Show editor state` button.
